sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/NuGetMetadataReference.cs

```diff
@@ -37,14 +37,14 @@ internal static class NuGetMetadataReference
         private static readonly List<string> allowedNugetLibDirectories =
             new List<string>
             {
-                "lib",
-                "portable-net45",
-                "net20",
-                "net40",
                 "net45",
-                "netstandard1.0",
+                "net40",
```

Comment:
Is this list now in priority order in which the first matching framework in a NuGet package will be chosen?
It would be helpful if the name/a comment made this obvious.